build_tools/kokoro/gcp_ubuntu/cmake/linux/x86-swiftshader-asan/build.sh

```diff
@@ -41,9 +41,14 @@ CMAKE_ARGS=(
 echo "Configuring CMake"
 "${CMAKE_BIN?}" "${CMAKE_ARGS[@]?}"
 
-echo "Building with CMake"
+echo "Building all"
+echo "------------"
 "${CMAKE_BIN?}" --build "${CMAKE_BUILD_DIR?}" -- -k 0
 
+echo "Building test deps"
+echo "------------------"
+"${CMAKE_BIN?}" --build "${CMAKE_BUILD_DIR?}" --target iree-test-deps -- -k 0
```

Comment:
I have a feeling that a few other scripts should also build this new `iree-test-deps` target.

@GMNGeoffrey can you review that part of this also?

Reply:
If they aren't in the CI, we'll just have to be on a voyage of self discovery on that. CTest does show a useful error on these situations (it says that a test input is not available and marks the test as "not run" and counts those as failures).

Reply:
Ah, they're in the _postsubmit_ CI:

* https://source.cloud.google.com/results/invocations/5908ee89-14b5-4432-9df4-fdb612ddf4ec/targets/iree%2Fgcp_ubuntu%2Fcmake%2Flinux%2Fx86-turing%2Fmain/log
* https://buildkite.com/iree/iree-android-arm64-v8a/builds/5827#3db40535-3a0d-4086-b4c0-17d64b4c21b1